### PR TITLE
Fix the flag used to get optional packages

### PR DESCRIPTION
**Pull request by gkhanna79**

### CODE REVIEW

**Comment by gkhanna79** [comment]
@jkotas PTAL

**Comment by jkotas** [comment]
Have you meant to include all the nuget cruft?

File: src/uwp/Host/UWPHost/HostEnvironment.cpp [hunk 1]
```diff
@@ -10,14 +10,9 @@
 //
 
 #include "windows.h"
+#include "appmodel.h"
 #include "HostEnvironment.h"
 
-// This should ideally come from WINDOWS SDK header but is missing in earlier versions
-#ifndef PACKAGE_FILTER_IS_IN_RELATED_SET 
-#define PACKAGE_FILTER_IS_IN_RELATED_SET 0x40000
```
Comment (jkotas) [inline src/uwp/Host/UWPHost/HostEnvironment.cpp]: I think you may need to keep this. The CI build machine does not seem to have Windows SDK with it.

**Comment by gkhanna79** [comment]
No, I didn't looks like it got in. I will clean it up.

File: src/uwp/Host/UWPHost/HostEnvironment.cpp (hunk 1 above)
Comment (gkhanna79) [inline src/uwp/Host/UWPHost/HostEnvironment.cpp]: I am trying to understand if I need to keep this - making few attempts before going back this way.

**Pull request closed without merging**